src/electron/window.main.ts

```diff
@@ -147,6 +151,15 @@ export class WindowMain {
         this.win.on('move', () => {
             this.windowStateChangeHandler(Keys.mainWindowSize, this.win);
         });
+
+    }
+
+    async alwaysOnTop(){
+
+        this.enableAlwaysOnTop = this.win.isAlwaysOnTop() ? false : true;
```

Comment:
I dont think you need the ternary operators here. Just use `this.enableAlwaysOnTop = !this.win.isAlwaysOnTop()`?